src/LiveComponent/assets/src/PollingDirector.ts

```diff
@@ -55,7 +55,7 @@ export default class {
             };
         }
 
-        const timer = setInterval(() => {
+        const timer = window.setInterval(() => {
```

Comment:
I am not sure about this fix. setInterval is native function well supported why use the one from window?

Reply:
Using `setInterval` or `window.setInterval` results in the same thing in a browser context. However, when using `setInterval`, TypeScript was mistakenly thinking that this was the Nodejs `setInterval` function, which doesn't return a number like in a browser, which was causing a typing error warning during builds.

Reply:
Ok thanks for your answer!